021-quarkus-panache-multiple-pus/src/main/java/io/quarkus/qe/multiplepus/VegetableResource.java

```diff
@@ -0,0 +1,86 @@
+package io.quarkus.qe.multiplepus;
+
+import io.quarkus.hibernate.orm.PersistenceUnit;
+import io.quarkus.panache.common.Sort;
+import io.quarkus.qe.multiplepus.model.vegetable.Vegetable;
+
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.transaction.Transactional;
+import javax.validation.Valid;
+import javax.ws.rs.ClientErrorException;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import java.util.List;
+
+@Path("vegetable")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+public class VegetableResource {
+
+    @Inject
+    @PersistenceUnit("vegetables")
+    EntityManager entityManager;
```

Comment:
EntityManager is never used. You are using PanacheEnityBase persist() and delete() methods